server/manager/src/main/java/org/apache/accumulo/manager/TabletGroupWatcher.java

```diff
@@ -398,6 +399,7 @@ private TableMgmtStats manageTablets(Iterator<TabletManagement> iter,
             "Error on TabletServer trying to get Tablet management information for extent: {}. Error message: {}",
             tm.getExtent(), mtiError);
         this.metrics.incrementTabletGroupWatcherError(this.store.getLevel());
```

Comment:
This line increments the metric that indicates that an error occurred in the TabletGroupWatcher.